common/app/rendering/core/JavascriptRendering.scala

```diff
@@ -56,7 +56,7 @@ trait JavascriptRendering extends Logging {
   private def prescript: ByteArrayInputStream = {
     val pre =
       """
-        |var global = global || this, self = self || this, window = window || this;
+        |var global = global || this, self = self || this, window = undefined;
```

Comment:
What happens when you just remove the window variable?

Reply:
yeah that's a more sensible idea!